man1/aur-fetch.1

```diff
@@ -0,0 +1,43 @@
+.TH AUR-FETCH 1 2018-02-14 AURUTILS
+.SH NAME
+aur-fetch \- fetches packages from a location
+
+.SH SYNOPSIS
+.SY aur
+.B fetch
+.OP -L log_dir
+.OP -m mode
+.YS
+
+.SH DESCRIPTION
+\fIaur-fetch\fR accepts package names from stdin, and attempts to
+download them. It understands two \fImodes\fR of operation; \fIgit\fR
+and \fItar\fR.
+
+\fIgit\fR attempts to download the package from a git repository.
+
+\fItar\fR attempts to downloade the tar archive with
+\fIaur-fetch-snapshot\fR.
+
+.SH OPTIONS
+.B \-L
+.RS
+The location of where the package should be downloaded.
```

Comment:
The location where build files are downloaded to is `$PWD`. `-L` is the location where _diffs_ (from `HEAD` before `git merge`) are stored. (example: https://github.com/AladW/aurutils/blob/master/lib/aur-sync#L264)